api/src/dev/server/Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN apt-get update && \
 
```

Comment:
Should probably bump the image version in docker-compose to invalidate everyone's  local builds too: https://github.com/all-of-us/workbench/blob/d0f2cb6cf0afef69efae437fe33d0535308c8453/api/docker-compose.yaml#L13